src/interventions/seeding.jl

```diff
@@ -19,17 +19,22 @@ function distribute_seeded_corals(
     available_space::Vector{Float64},
     seeded_area::YAXArray
 )::YAXArray
+    total_seeded_area::Float64 = sum(seeded_area)
+    total_available_space::Float64 = sum(available_space)
 
     # Proportion of available space on each site relative to available space at these
     # locations
-    prop_area_avail = available_space ./ sum(available_space)
+    prop_area_avail = available_space ./ total_available_space
+    if total_seeded_area > total_available_space
+        @warn "Seeded area exceeds available space. Restricting to available space."
```

Comment:
I think worthwhile to include the available area and the seeded area to give an indication of the magnitude of the problem.